client/components/feature-item/index.tsx

```diff
@@ -18,28 +19,36 @@ const FeatureItemContainer = styled.div`
 	}
 `;
 
-const FeatureItemHeader = styled.div`
+interface FeatureItemHeaderProps {
+	dark?: boolean;
+}
+
+interface FeatureItemContentProps {
+	dark?: boolean;
+}
+
+const FeatureItemHeader = styled.div< FeatureItemHeaderProps >`
 	margin-bottom: 16px;
 	font-size: var( --scss-font-body );
 	font-weight: 500;
 	line-height: 24px;
-	color: var( --color-text-inverted );
+	color: var( --${ ( props ) => ( props.dark ? 'color-text-inverted' : 'color-text' ) } );
```

Comment:
This was needed because removing the `dark` prop from the parent `Section` results in white-on-white Feature Item headers.